packages/react-native-editor/src/test/index.test.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * External dependencies
+ */
+import { AppRegistry } from 'react-native';
+// eslint-disable-next-line import/no-unresolved
+import { render } from 'test/helpers';
```

Comment:
I had to disable `import/no-unresolved` rule for this import statement because it's not recognized by the import resolver ([reference](https://github.com/WordPress/gutenberg/blob/trunk/tools/eslint/import-resolver.js)).